app/experimenter/experiments/tests/api/v4/test_serializers.py

```diff
@@ -36,10 +41,81 @@ def test_serializer_outputs_expected_schema(self):
         ExperimentVariantFactory.create(experiment=experiment, slug="variant-2")
 
         serializer = ExperimentRapidRecipeSerializer(experiment)
-        data = serializer.data
+        data = serializer.data.copy()
 
-        fn = os.path.join(os.path.dirname(__file__), "experimentRecipe.json")
```

Comment:
And I got rid of the schema validation in the DRAFT test because a bunch of fields aren't present before the experiment launches and that doesn't match the schema, so either we:

- update the schema to allow for those fields to be `null`able before launch
- don't emit a rapid recipe at all for draft experiments
- emit a rapid recipe for draft experiments with those fields set to `null` and then just say it doesn't need to validate against the schema, which is the option I went for here, and the fully completed recipe for the review state can be validated against the schema

Since I think we'll be moving all of that into the Experimenter repo and out of the shared repo, we can re evaluate how we handle this after that happens.